core/src/CMakeLists.txt

```diff
@@ -50,28 +42,47 @@ set(SOURCES_IMPL
   impl/Cabana_Version.cpp
   )
 
-#-----------------------------------------------------------------------------
-
-install(FILES ${HEADERS_PUBLIC} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
-install(FILES ${HEADERS_IMPL} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/impl/)
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/CabanaCore_config.hpp DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
-
-#-----------------------------------------------------------------------------
-
 add_library(cabanacore ${SOURCES_IMPL})
-target_include_directories(cabanacore PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
-  $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}> $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
+
 target_link_libraries(cabanacore Kokkos::kokkos)
-if(Cabana_ENABLE_ARBORX)
+
+if(ARBORX_FOUND)
   target_link_libraries(cabanacore ArborX::ArborX)
 endif()
-if(Cabana_ENABLE_MPI)
+
+if(MPI_FOUND)
   target_link_libraries(cabanacore MPI::MPI_CXX)
 endif()
-install(TARGETS cabanacore EXPORT Cabana_Targets LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR} ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
-install(EXPORT Cabana_Targets FILE Cabana_Targets.cmake NAMESPACE Cabana:: DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/Cabana)
+
+target_include_directories(cabanacore
+  PUBLIC
+  $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+  $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>
+  $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
+
+install(TARGETS cabanacore
+  EXPORT Cabana_Targets
+  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
+
+install(EXPORT Cabana_Targets
+  FILE Cabana_Targets.cmake
+  NAMESPACE Cabana::
+  DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/Cabana)
+
+install(FILES "${CMAKE_CURRENT_BINARY_DIR}/CabanaConfig.cmake" "${CMAKE_CURRENT_BINARY_DIR}/CabanaConfigVersion.cmake"
+  DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/Cabana)
+
+install(FILES ${HEADERS_PUBLIC} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+
+install(FILES ${HEADERS_IMPL} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/impl/)
+
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/CabanaCore_config.hpp DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+
 include(CMakePackageConfigHelpers)
-configure_file(CabanaConfig.cmakein ${CMAKE_CURRENT_BINARY_DIR}/CabanaConfig.cmake @ONLY)
-write_basic_package_version_file("CabanaConfigVersion.cmake" VERSION ${PROJECT_VERSION} COMPATIBILITY ExactVersion)
-install(FILES "${CMAKE_CURRENT_BINARY_DIR}/CabanaConfig.cmake" "${CMAKE_CURRENT_BINARY_DIR}/CabanaConfigVersion.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/Cabana)
-add_library(Cabana::cabanacore ALIAS cabanacore)
```

Comment:
@sslattery you will to re-add this alias to fix travis.

Reply:
Oh crap forgot about that - yeah that was a mistake to remove. Good catch